gtsfm/evaluation/compare_metrics.py

```diff
@@ -0,0 +1,60 @@
+"""Script to log metrics for comparison across SfM pipelines (GTSfM, COLMAP, etc.) as a JSON.
+
+Authors: Jon Womack
+"""
+import os
+import json
+from typing import Dict
+
+from gtsfm.evaluation.metrics import GtsfmMetric, GtsfmMetricsGroup
+
+
+
+def compare_metrics(txt_metric_paths: Dict[str, str], json_path: str) -> None:
+    """
```

Comment:
missing 1-line docstring/explanation

Reply:
Added.

Reply:
Yes, fixed.

Reply:
nit: Please leave a line between the description and args, and between args and returns. 
```
"""blah blah

Args:
  foo

Returns:
  bar
"""
```

Reply:
same in other places